py/src/braintrust/logger.py

```diff
@@ -1743,6 +1770,11 @@ def __exit__(self, type, value, callback):
         self.end()
 
 
+def add_dataset_output_alias(r):
+    r["output"] = r.get("expected")
```

Comment:
If you happened to receive data via the legacy endpoint, then wouldn't `output` be set and this would override it to `None`?

Reply:
Should we just use

```
def patch_legacy_dataset(r):
    row = {**r}
    output = row.pop("output", None)
    if "expected" not in row:
        row["expected"] = output
    return row
```

here?